screens/ChatScreen.js

```diff
@@ -71,11 +70,18 @@ class ChatScreen extends Component {
     }
 
     this.setState({ loading: false });
+
+    // If we are the user who created the channel, always attempt to invite the other user
+    //  We only want to do this if we created the channel, because any other user on the channel
+    //   will not have permission to invite new users
+    if (this.channel.createdBy === this.state.id) {
+      this.tryInviteContact();
```

Comment:
Yeah so there's three steps: 

1. Create a twilio user for a user in our db (will happen on backend when we add new users)
2. Create a twilio channel for chat (will happen by whoever signs into app first)
3. Invite other user(s) to channel (only the user who created the channel can do this. We will always attempt to invite a user every time the user who created the channel signs on). 

This check just ensures that (3) only happens by the user who created the channel. We should be able to modify this step once we have the user init logic in place, because an invite should stay in place as long as the user being invited exists.

Reply:
Would this user init logic live on the client or the server?

Reply:
I think we'll ultimately need it server-side, because we need the user and all it's contacts to exist before the first user signs in (otherwise we potentially end up with the user making the channel and attempting to invite a non-existent user).